src/term/transform/encode_pattern_matching.rs

```diff
@@ -354,6 +375,39 @@ impl Term {
   fn arg_call(term: Term, arg: Name) -> Term {
     Term::App { tag: Tag::Static, fun: Box::new(term), arg: Box::new(Term::Var { nam: arg }) }
   }
+
+  fn optional_arg_call(term: Term, arg: Option<Name>) -> Term {
+    let arg = Box::new(arg.map_or(Term::Era, |nam| Term::Var { nam }));
+    Term::App { tag: Tag::Static, fun: Box::new(term), arg }
+  }
+
+  // Returns a sequence representing whether the lambda arg is binded or ignored
+  fn lambdas_usage(&self) -> Vec<Option<()>> {
```

Comment:
This function name is a bit hard to understand. Something like `arg_vars_are_used` maybe? Not sure, this one is a bit hard.